chalicelib/checks/wfr_checks.py

```diff
@@ -844,30 +1179,34 @@ def SV_metawfrs_to_patch_sample_processing(connection, **kwargs):
                     SV_processed += 1
                 elif pf['variant_type'] == "SNV":
                     SNV_processed += 1
-            except:
+            except Exception:
```

Comment:
Maybe not something for now but perhaps consider defining some exception types, just simply giving a name and a description can be very helpful when tracking down issues. Might be you want to do this in dcicutils.

Reply:
I agree with that, definitely something to do as we keep refactoring things